praw/models/reddit/subreddit.py

```diff
@@ -1548,6 +1548,36 @@ def __call__(self, id=None, mark_read=False):
 
            reddit.subreddit('redditdev').modmail('2gmz', mark_read=True)
 
+        To print all messages from a conversation as Markdown source:
+
+        .. code:: python
+
+           conversation = reddit.subreddit('redditdev').modmail('2gmz',
+                                                                mark_read=True)
+           for message in conversation.messages:
+               print(message.body_markdown)
+
+        ``ModmailConversation.user`` is a special instance of
```

Comment:
As I mentioned [here](https://www.reddit.com/r/redditdev/comments/6aihxq/praw_bug_modmailconversationsstateall_not/dhfx0s4/), there's a tenacious bug on Reddit's end whereby the `user` attribute can fail to exist. I considered documenting this here, so that users can catch the `AttributeException` if relevant.